migrations/Version202009241529173772_taoLti.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoLti\models\classes\LtiProvider\FeatureFlagFormPropertyMapper;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202009241529173772_taoLti extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return '';
+    }
+
```

Comment:
```suggestion
```
Can be removed, we do not need it.

Reply:
This is part of migration class. I added description

Reply:
This is not required. You can delete the method (it is already implemented in the parent class). IMO the description are one more step to maintain that we did not have before (using Updater). But I am closing. It is up to you